packages/modules/manage-versions/src/components/contextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SvgMore } from "@itwin/itwinui-icons-react";
+import { DropdownMenu, IconButton, MenuItem } from "@itwin/itwinui-react";
+import React from "react";
+
+export type MenuAction = {
+  icon?: JSX.Element;
+  onClick: () => void;
+  title: string;
+  label: string;
+  disabled?: boolean;
+};
+
+export interface ContextMenuProps {
+  menuActions: MenuAction[];
+  isMenuOpen: boolean;
+  toggleMenu: (rowId: string) => void;
+  rowId: string;
+  onClose: () => void;
+}
+
+export const ContextMenu = ({
+  menuActions,
+  isMenuOpen,
+  toggleMenu,
+  rowId,
+  onClose,
+}: ContextMenuProps) => {
+  return (
+    <DropdownMenu
+      onClickOutside={onClose}
+      menuItems={() =>
+        menuActions.map((action, index) => (
+          <MenuItem
+            {...action}
```

Comment:
Spreading is not a good idea.. you add not valid props and get no validation (i.e. `label` goes there too)

Reply:
@veekeys Yes, my bad, I have removed spread opertaor and passing alone what is needed now.